src/opentracing_external.cpp

```diff
@@ -21,18 +24,20 @@ std::tuple<std::shared_ptr<ot::Tracer>, std::shared_ptr<TraceEncoder>> makeTrace
     const TracerOptions &options) {
   auto maybe_options = applyTracerOptionsFromEnvironment(options);
   if (!maybe_options) {
-    std::cerr << "Error applying TracerOptions from environment variables: "
-              << maybe_options.error() << std::endl
-              << "Tracer will be started without options from the environment" << std::endl;
+    std::ostringstream message;
+    message << "Error applying TracerOptions from environment variables: " << maybe_options.error()
+            << "\nTracer will be started without options from the environment\n";
+    StandardLogger(options.log_func).Log(LogLevel::error, message.str());
```

Comment:
If the tracer isn't the part that creates the logger but it's injected, then maybe makeLogger earlier and just use the one it made?